tests/end_to_end/server_test.py

```diff
@@ -48,11 +46,7 @@ def run(self, *args, **kwargs):
             self.otherTeardown()
 
     def runClientCmd(self, client, command):
-        try:
-            client.runCommand(command)
-        except subprocess.CalledProcessError as error:
-            self.server.printDebugInfo()
-            raise error
+        client.runCommand(command)
```

Comment:
"spam" is output that could not be useful in any context whatsoever.  It is quite useful if, upon failure, a test prints out information that helps correct errors in the code.